app/Filament/App/Pages/Settings.php

```diff
@@ -0,0 +1,292 @@
+                                    ])
+                                    ->autosize()
+                                    ->live(onBlur: true)
+                                    ->formatStateUsing(fn () => $server->description)
+                                    ->afterStateUpdated(function ($state) {
+                                        $data = [
+                                            'description' => $state ?? '',
+                                        ];
+                                        $this->updateDescription($data);
+                                    }),
+
+                                TextInput::make('uuid')
+                                    ->label('Server UUID')
+                                    ->columnSpan([
+                                        'default' => 1,
+                                        'sm' => 1,
+                                        'md' => 3,
+                                        'lg' => 5,
+                                    ])
+                                    ->disabled()
+                                    ->formatStateUsing(fn () => $server->uuid),
+                                TextInput::make('id')
+                                    ->label('Server ID')
+                                    ->disabled()
+                                    ->formatStateUsing(fn () => $server->id)
+                                    ->columnSpan(1),
+
+                            ]),
+                        Fieldset::make('Limits')
+                            ->label('Limits')
+                            ->columns([
+                                'default' => 1,
+                                'sm' => 1,
+                                'md' => 3,
+                                'lg' => 3,
+                            ])
+                            ->schema([
+                                TextInput::make('backup_limit')
+                                    ->label('Backup Limit')
+                                    ->columnSpan(1)
+                                    ->disabled()
+                                    ->formatStateUsing(fn () => $server->backups->count() . ' of ' . $server->backup_limit),
+                                TextInput::make('database_limit')
+                                    ->label('Database Limit')
+                                    ->columnSpan(1)
+                                    ->disabled()
+                                    ->formatStateUsing(fn () => $server->databases->count() . ' of ' . $server->database_limit),
+                                TextInput::make('allocation_limit')
+                                    ->label('Allocation Limit')
+                                    ->columnSpan(1)
+                                    ->disabled()
+                                    ->formatStateUsing(fn () => $server->allocations->count() . ' of ' .  $server->allocation_limit + 1),
```

Comment:
```suggestion
                                    ->formatStateUsing(fn () => $server->allocations->count() . ' of ' .  $server->allocation_limit ?? 0 + 1),
```
https://github.com/pelican-dev/panel/blob/038eca8c4b90324517dd0fe678d442e8ed44fa24/app/Models/Server.php#L45

Reply:
Please don't make reviews to fix linting issues.
We are constantly working on the branch and all linting issues will be resolved eventually.